EPNTAP.tex

```diff
@@ -1832,17 +1839,19 @@ \subsubsection{Solar System Objects extension}
 
 Solar system sidereal rotation period (in hour)
 
-\paragraph{semi\_major\_axis}
+\paragraph{semi\_major\_axis, inclination, eccentricity, long\_asc, arg\_perihel, mean\_anomaly}
 
-in au,
-
-inclination, etc
+Provide the standard orbital parameters (distance in AU, angles in degrees)
 
 See discussion page: orbital
 parameters\footnote{\url{https://voparis-wiki.obspm.fr/display/VES/orbital+parameters}}
 
+\paragraph{magnitude, flux, albedo}
+Provide values of respective quantities (flux in mJy).
+
+
 
-\textbf{\\}When target\_class\textbf{ = asteroid or dwarf\_planet},
+\textbf{\\}When target\_class\textbf{ = asteroid, dwarf\_planet or comet},
```

Comment:
Why is there a `\textbf{\\}` at the beginning? This should be cleaned up I guess.

Reply:
OK